modules/metrics.py

```diff
@@ -1,7 +1,28 @@
 import enum
-
+import functools
 import prometheus_client
+import logging
+
+
+class ExitStatus(enum.Enum):
+    SUCCESS = (0, "success")
+    GENERIC_ERROR = (1, "generic_error")
+    INVALID_ARGUMENT = (2, "invalid_argument")
+    OUT_OF_MEMORY = (137, "out_of_memory")
+    SEGMENTATION_FAULT = (139, "segmentation_fault")
+    KILL = (-9, "kill")
```

Comment:
what is the return code of the ffmpeg process when we stop the video from the browser? it might not be -9, i would check locally

Reply:
it was -9 from my end